src/electron/ipc-api/dnd.ts

```diff
@@ -19,7 +19,7 @@ export default async () => {
     try {
       const isDND = getDoNotDisturb();
       debug('Fetching DND state, set to', isDND);
-      return isDND;
+      return await isDND;
```

Comment:
The correct way would be to do `const isDND = await getDoNotDisturb()` and then keep `return isDND;`

The debug statement above would just output "pending promise" the way it is written now.

Reply:
You are right, sorry I didn't see that it `getDoNotDisturb` has been changed as a promise, instead of directly returning a boolean as it did before and the change was made automatically by the compiler.